tools/nl/embeddings/data/curated_input/base/sheets_svs.csv

```diff
@@ -2511,7 +2497,7 @@ Count_Person_NonWorker_Urban,Number of urban people who do not work
 Count_Person_NotAUSCitizen,Number of non US citizens
 Count_Person_NotEnrolledInSchool,Number of people not enrolled in school
 Count_Person_NotInLaborForce,Number of people Aged 16 and Over Not in the Labor Force
-Count_Person_NowMarried,Number of married females
+Count_Person_NowMarried,Number of married (including separated) people

```

Comment:
nit: "Number of married people, including separated"

Not sure how having something in `()` affects

Reply:
okay, i will update in following PR, so not re-generate the embeddings again.

Reply:
makes sense!